operations/mimir-mixin/dashboards/tenants.libsonnet

```
@@ -530,5 +530,83 @@ local filename = 'mimir-tenants.json';
         ) +
         { legend: { show: false } },
       )
-    ),
+    )
+
+    .addRow(
+      $.row('Read Path - Insights')
+      .addPanel(
+        local title = 'Rate of Read Requests';
+        $.panel(title) +
+        $.queryPanel(
+          'sum(rate(cortex_query_frontend_queries_total{%s, user="$user"}[$__rate_interval]))' % $.namespaceMatcher(),
```

We should match the container, because we have another deployment called `ruler-query-frontend`

```suggestion
          'sum(rate(cortex_query_frontend_queries_total{%s, container="query-frontend", user="$user"}[$__rate_interval]))' % $.namespaceMatcher(),
```